openapi/newsapi/Dependencies.toml

```diff
@@ -1,11 +1,9 @@
 [[dependency]]
 org = "ballerina"
 name = "http"
-version = "1.1.0-alpha8"
+version = "1.1.0-beta.2"
 
 [[dependency]]
 org = "ballerina"
 name = "url"
-version = "1.1.0-alpha8"
-
-
+version = "1.1.0-beta.2"
```

Comment:
Please stick to existing dependency versions until Choreo migrating to Ballerina beta.